tools/gui/qtkontakt.py

```diff
@@ -725,6 +741,91 @@ def __set_uhrzeit_datum(self, uhrzeit: str, widget: QtWidgets.QTimeEdit):
         assert(QTime.isValid(time))
         widget.setTime(time)
 
+    ##############################
+    #        Notifications       #
+    ##############################
+
+    def __get_notifications(self) -> dict:
+        """
+        Ruft Werte für die Benachrichtigungen aus der GUI ab
+        und gibt diese als Dict zurück.
+
+        Returns:
+            Dict mit den notifications Werten
+        """
+
+        # Pushover
+        app_token = self.i_app_token.text().strip()
+        user_key = self.i_user_key.text().strip()
+
+        # Telegram
+        api_token = self.i_api_token.text().strip()
+        chat_id = self.i_chat_id.text().strip()
+
+        notifications = {}
+
+        if app_token != "" and user_key != "":
+            notifications['pushover'] = {}
+            notifications['pushover']['app_token'] = app_token
+            notifications['pushover']['user_key'] = user_key
+
+        if api_token != "" and chat_id != "":
+            notifications['telegram'] = {}
+            notifications['telegram']['api_token'] = api_token
+            notifications['telegram']['chat_id'] = chat_id
+
+        return notifications
+
+    def __set_notifications(self, notifications: dict):
+        """
+        Werte aus übergebenem Dict werden in die passenden QLineEdits geschrieben
+
+        Args:
+            notifications (dict): Enthält pushover und telegram spezifische Werte
+        """
+
+        if 'pushover' in notifications:
+            self.i_app_token.setText(notifications['pushover']['app_token'])
+            self.i_user_key.setText(notifications['pushover']['user_key'])
+
+        if 'telegram' in notifications:
+            self.i_api_token.setText(notifications['telegram']['api_token'])
+            self.i_chat_id.setText(notifications['telegram']['chat_id'])
+
+    def __reset_notifications(self):
+        """
+        Setzt alle Werte für die Benachrichtigungen (notifications) in der GUI zurück
+        """
+
+        for widget in self.notifications_tab.children():
+            if isinstance(widget, QtWidgets.QLineEdit):
+                widget.setText("")
```

Comment:
Du kannst hier auch direkt nur nach QLineEdit suchen 
```suggestion
        for line_edit in self.notifications_tab.findChildren(QtWidgets.QLineEdit):
                line_edit.setText("")
```